### DISCUSSION TITLE
Outline with Keycloak SSO

### DISCUSSION THREAD

**Discussion by Zlateshkin1337** [outline/outline#6636] [Self Hosting]
Hello, I deployed a wiki using docker compose and integrated Keycloak. The caseload itself sets the session time and token lifetime to one week. But even with this configuration, users are logged out several times during the day. In the wiki itself, I did not find a configuration for the session time. help me please.

**Comment by tommoor** [comment]
Outline regularly pings the authentication provider to verify the token is still valid, attempting to load the `user.info` endpoint. If you're being logged out it is because this check is failing. Perhaps the server cannot contact your OIDC server? This can be a DNS issue.

Look for error logs on the server.

**Reply by Zlateshkin1337** [reply]
Impossible. The token's lifetime is a week; there can be no problems with DNS, since Keycloak is accessed via an external domain name

**Reply by tommoor** [reply]
I would suggest reading this thread:
https://github.com/outline/outline/issues/5564#issuecomment-1772799949
